Source/JavaScriptCore/wasm/WasmTypeDefinition.h

```diff
@@ -59,6 +59,163 @@ enum class ExtSIMDOpType : uint8_t {
 };
 #undef CREATE_ENUM_VALUE
 
+constexpr std::pair<size_t, size_t> countNumberOfWasmExtendedSIMDOpcodes()
+{
+    uint8_t numberOfOpcodes = 0;
+    uint8_t mapSize = 0;
+#define COUNT_EXT_SIMD_OPERATION(name, id, ...) \
+    numberOfOpcodes++; \
+    mapSize = std::max<size_t>(mapSize, (size_t)id);
+    FOR_EACH_WASM_EXT_SIMD_OP(COUNT_EXT_SIMD_OPERATION)
+#undef COUNT_EXT_SIMD_OPERATION
+    return { numberOfOpcodes, mapSize + 1 };
+}
+
+constexpr bool isRegisteredWasmExtendedSIMDOpcode(ExtSIMDOpType op)
+{
+    switch (op) {
+#define CREATE_CASE(name, id, ...) case ExtSIMDOpType::name:
+    FOR_EACH_WASM_EXT_SIMD_OP(CREATE_CASE)
+#undef CREATE_CASE
+        return true;
+    default:
+        return false;
+    }
+}
+
+constexpr void dumpExtSIMDOpType(PrintStream& out, ExtSIMDOpType op)
```

Comment:
I think there should already be a dump() for this, but if now we should make the standard WTF dump here